### DISCUSSION TITLE
blog/2023-01-28-sudo-with-ssh-keys

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#37] [vinnie.work Comments]
# blog/2023-01-28-sudo-with-ssh-keys

Background

https://www.vinnie.work/blog/2023-01-28-sudo-with-ssh-keys

**Comment by gaku** [comment]
Great post!  Thank you for sharing.  I've done the same with my server.

**Comment by addap** [comment]
Thanks for the post. I referenced it while setting this up on my server.
One correction: You don't need to copy the private key to the server. The point of agent forwarding is that the private key remains on your local machine but you can still authenticate with it via the forwarded agent.
